### PR TITLE
Fix #121 : require explicitly supported content types

### LINKED ISSUE #121: Link to developer files when dependency is bundled
Hello !

Thank you for the time and effort you put in this project :)
We used your project recently and we encountered an issue when starting our node server :
```
~/dev/my_project/release • node server.js
fs.js:856
  return binding.readdir(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, scandir '/Users/abalhier/dev/my_project/node_modules/busboy/lib/types'
    at Error (native)
    at Object.fs.readdirSync (fs.js:856:18)
    at Object.<anonymous> (/Users/abalhier/dev/my_project/release/server.js:10960:4)
    at Object.__dirname.13.module.exports.extractFormDataInputHierachy.28../utils (/Users/abalhier/dev/my_project/release/server.js:10972:4)
    at s (/Users/abalhier/dev/my_project/release/server.js:1:316)
    at /Users/abalhier/dev/my_project/release/server.js:1:367
    at Object.__dirname.13.module.exports.objectFromHierarchyArray.busboy (/Users/abalhier/dev/my_project/release/server.js:1614:16)
    at s (/Users/abalhier/dev/my_project/release/server.js:1:316)
    at /Users/abalhier/dev/my_project/release/server.js:1:367
    at Object.__dirname.13.module.exports.extractFormDataInputHierachy.252.exports.254.257.passport.263.623.../authentication/authentication (/Users/abalhier/dev/my_project/release/server.js:118103:20)
```

We are bundling dependencies and our code with node-browserify which may not be the traditional way to deploy and run node servers, but we feel this issue could be resolved easily by removing the use of __dirname.
Can you take a look at the pull request I am submitting ?

### PR DESCRIPTION
**Pull request by punkeel**
Make it easier to bundle `busboy`.
Original PR: https://github.com/mscdex/busboy/pull/122

### CODE REVIEW

**Comment by punkeel** [comment]
fixed the lint issues

File: lib/index.js [hunk 1]
```diff
@@ -37,14 +34,10 @@ function getInstance(cfg) {
...
 
-const TYPES = [];
-for (const file of readdirSync(join(__dirname, 'types'))) {
-  if (/\.js$/i.test(file)) {
-    const type = require(`./types/${file}`);
-    if (type && typeof type.detect === 'function')
-      TYPES.push(type);
-  }
-}
+const TYPES = [
```
Comment (mscdex) [inline lib/index.js:37]:
```suggestion
// Note: types are explicitly listed here for easier bundling
// See: https://github.com/mscdex/busboy/issues/121
const TYPES = [
```

**Comment by mscdex** [comment]
Landed in 5e4072cf7d7e0d7f406639182777642c3419ec41. Thanks!

**Comment by punkeel** [comment]
Thank you for accepting & tagging a release! 🙏

**Pull request closed without merging**